host/host.go

```diff
@@ -1,3 +1,6 @@
+// Package host provides the core Host interface for libp2p.
+//
+// Host represents a single libp2p "agent" that can act as client or server.
```

Comment:
Client/server terminology is a wee bit confusing in p2p.

```suggestion
// Host represents a single libp2p node in a peer-to-peer network.
```